src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -235,7 +235,7 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
     );
 
   const updateAlumniInfoButton =
-    profile.PersonType === 'alu' && isOnline && myProf ? (
+    true && isOnline && myProf ? (
```

Comment:
Also here

Reply:
I'm going to make a minor adjustment to make it check for isAlumni. I don't know why it wasn't that in the first place.